package.json

```diff
@@ -33,9 +34,6 @@
   "engines": {
     "node": ">=10"
   },
-  "lint-staged": {
-    "*.{js,ts,sol}": "prettier --write"
-  },
```

Comment:
Yeah my thoughts around not including it are two fold - I think if it's unexpected it can have unintended consequences but this is a minor issue and shouldn't be a problem unless we have community contributions. Beyond that the auto fix on `solhint` is mediocre at best without prettier, oftentimes not fixing basic issues. The auto fix on `forge fmt` is pretty good. I'll add it back in but we will just need to be vigilant in making sure it actually behaves as expected. Same with the GitHub action.